### PR TITLE
Teams tab view with team members (includes invites)

### PR DESCRIPTION
**Pull request by katiearriagam**
## Changes
- Tabs layout for team details view
- Includes 'Members' tab which shows team members and the dropdown to add new members to the team.

## Run this PR

- **IMPORTANT:** Run `yarn install`  (and then compile with `yarn run prod` or `yarn run frontend`) to run this PR

## Partially fixes
- #27

### CODE REVIEW

File: webpack.mix.js [hunk 1]
```diff
@@ -12,7 +12,8 @@ const mix = require('laravel-mix');
  */
 
 mix.js('resources/js/app.js', 'public/js')
-.js('resources/js/searchFunction.js', 'public/js')
+.js('resources/js/utilities.js', 'public/js')
+.js('resources/js/searchProjects.js', 'public/js')
```
Comment (hecerinc) [inline webpack.mix.js]: El searchFunction creo que es el que se queda, no el otro. @marcegarzab  please confirm

Comment (hecerinc) [reply webpack.mix.js]: De hecho no puedo correr esto por este bug, creo que si es el otro.

File: resources/views/teams/details.blade.php [hunk 2]
```diff
@@ -4,33 +4,207 @@
...
+								'projectShortDescription' => $project->short_description,
+								'labels' => $project->labels,
+								'publicMilestone' => 'shipping'
+							])
+							@endprojectCard
+						@endforeach
+					</div>
+				@else
+					<div id="no-projects-msg" class="ui message">
+						<p class="header">
```
Comment (hecerinc) [inline resources/views/teams/details.blade.php]: Nit: can we put this into a single line?

File: resources/views/teams/details.blade.php [hunk 3]
```diff
@@ -4,33 +4,207 @@
...
+							No projects found!
+						</p>
+						<p>Looks like this team does not own any projects... yet!</p>
+					</div>
+				@endif
+			</div>
+			<div class="ui bottom attached tab segment" data-tab="members">
+				<div class="{{ count($team->members) <= 0 ? 'hidden' : '' }}">
```
Comment (hecerinc) [inline resources/views/teams/details.blade.php]: There's no case where a team member count is 0, this is superfluous

File: resources/views/teams/details.blade.php [hunk 4]
```diff
@@ -4,33 +4,207 @@
...
+		/* render sent datetime for all invites*/
+		renderDateTimeAgoOnce();
+
+		/* Validate invitation to join the team */
+		function validateNewMember() {
+			// get the info of the user that will receive the invitation
+			const newMemberName = $("#new-member-dropdown").dropdown("get text");
+			const newMemberId = $("#new-member-dropdown").dropdown("get value");
+
+			if(newMemberId === $("#new-member-dropdown").dropdown("get default value")) {
```
Comment (hecerinc) [inline resources/views/teams/details.blade.php]: El default value tmb deber&iacute;as separarle en su propia variable, this looks very ugly

File: resources/views/teams/details.blade.php [hunk 5]
```diff
@@ -4,33 +4,207 @@
...
 <div class="padded content">
-	<h1>{{ $team->name }}</h1>
-	<div class="team-info">
-		<div class="image-container">
-			<div class="image-uploader preview-container">
-				<img id="preview" src="{{ isset($team->img_url) ? $team->img_url : 'https://dummyimage.com/400x400/3498db/ffffff.png&text=B' }}" alt="Team image" class="ui small image preview"/>
+	<div class="ui stackable grid">
+		<div class="four wide column">
+			<h1 style="text-align: center;">{{ $team->name }}</h1>
+			<img src="{{ isset($team->img_url) ? $team->img_url : 'https://dummyimage.com/400x400/3498db/ffffff.png&text=B' }}" alt="Team image" class="ui small image preview" style="margin: 0 auto; margin-bottom: 20px; border-radius: 15px;"/>
```
[the remark this answers is not in the document]
Comment (katiearriagam) [reply resources/views/teams/details.blade.php]: Ya abrí un task para esto #193. Haremos los cambios en un PR separado porque hay que cambiarlo en varias partes (nav bar, user cards, tables, etc.)

File: resources/views/teams/details.blade.php [hunk 6]
```diff
@@ -4,33 +4,206 @@
...
+			if(newMemberId === defaultValue) {
+				$("#new-member-dropdown").addClass("error");
+				return false;
+			}
+
+			$("#newMemberName").text(newMemberName);
+			$("#member-to-add-modal").modal("show");
+		}
+
+		/* Generate a table row with the info of the user to inveite */
```
Comment (melytc) [inline resources/views/teams/details.blade.php]: typo: invite 😛

File: resources/views/projects.blade.php [hunk 7]
```diff
@@ -49,6 +49,19 @@
 
 @section('scripts')
 <script>
+	function adjustProjectsGrid(maxWidth) {
```
Comment (hecerinc) [inline resources/views/projects.blade.php]: Big nono. El CSS framework hace esto por ti:

https://semantic-ui.com/collections/grid.html#doubling o en todo caso los "wide device" que vienen ah&iacute; abajo

Comment (katiearriagam) [reply resources/views/projects.blade.php]: oh cool - totally missed that. thanks!

**Pull request merged**